src/librustdoc/html/static/main.js

```diff
@@ -349,28 +349,35 @@ function defocusSearchBar() {
         }
     }
 
-    function getHelpElement() {
-        buildHelperPopup();
+    function getHelpElement(build) {
+        if (build !== false) {
+            buildHelperPopup();
+        }
         return document.getElementById("help");
     }
 
     function displayHelp(display, ev, help) {
-        help = help ? help : getHelpElement();
         if (display === true) {
+            help = help ? help : getHelpElement();
```

Comment:
Oh this makes me so sad :( I know it doesn't affect the behavior, but can you pass an argument anyway?
```suggestion
            help = help ? help : getHelpElement(true);
```

Reply:
Oh sure.